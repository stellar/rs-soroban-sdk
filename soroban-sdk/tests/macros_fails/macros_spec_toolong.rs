use soroban_sdk::contractimpl;

pub struct Contract;

#[contractimpl]
impl Contract {
    pub fn pay_to_destionation() {
        unimplemented!()
    }
}

pub fn main() {}
