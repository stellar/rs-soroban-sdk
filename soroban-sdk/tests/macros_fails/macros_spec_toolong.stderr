error: custom attribute panicked
 --> tests/macros_fails/macros_spec_toolong.rs:5:1
  |
5 | #[contractimpl]
  | ^^^^^^^^^^^^^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: LengthExceedsMax
